README.md

```diff
@@ -484,22 +484,33 @@ $ tfwrapper plan
 
 Terraform stacks are organized based on their:
 
-- account: an account alias which may reference one or multiple providers accounts. `aws-production`, `azure-dev`, etc…
-- environment: `production`, `preproduction`, `dev`, etc…
-- region: `eu-west-1`, `westeurope`, `global`, etc…
-- stack: defaults to `default`. `web`, `admin`, `tools`, etc…
+- `account`: an account alias which may refer to provider accounts or subscriptions, e.g. `project-a-prod`, `customer-b-dev`.
+- `environment`: `production`, `preproduction`, `dev`, etc. With `global` as a special case eliminating the `region` part.
+- `region`: `eu-west-1`, `westeurope`, etc.
+- `stack`: defaults to `default`. `web`, `admin`, `tools`, etc.
```

Comment:
This from https://github.com/claranet/tfwrapper/issues/246#issuecomment-1754433802

> does not contain `/` or `_` characters.

actually is an undocumented requirement.
Being ANAL here: it is left to be deduced by a reader from the `${account}_${environment}_${region}_${stack}.yml` and the fact that if `_`-s are used as separators, then *likely* those can't be used in names.